vendor/engines/split_accounts/app/assets/javascripts/split_account_splits.js.coffee

```diff
@@ -0,0 +1,17 @@
+# Update the `percent` value in each visible `split` so the total of all splits
+# adds up to 100.
+updateSplits = (event, param) ->
+  switch param.object_class
+    when "split", "split_accounts/split"
+      $container = $(event.target).closest("[data-subaccounts]")
+      $inputs = $container.find("[data-percent]").filter(":visible")
+
+      if $inputs.length > 0
+        percent = Math.round((100.0 / $inputs.length) * 100) / 100
+        remainder = Math.round((100.0 - percent * ($inputs.length - 1)) * 100) / 100
+        $inputs.val(percent)
+        $inputs.last().val(remainder)
+
+# Register event listeners when nested_form_fields are added or removed
+$(document).on "fields_added.nested_form_fields", updateSplits
```

Comment:
As of right now, I only load this code if the related view is loaded; it's not included in the javascript application manifest.  Alternatively, we could add it to the manifest and using the `.js--` prefix (or better yet a html5 `data` attribute) to trigger it, but I wonder since this is so rarely used if it's better to load it on demand?